src/App.js

```diff
@@ -1,13 +1,17 @@
 import { Home, Profile } from "./pages";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom"
+import ErrorPage from "./pages/ErrorPage.jsx";
```

Comment:
We can import `ErrorPage` similar to `Home` and `Profile` page from `./pages` directory :)
```js
import { Home, Profile, ErrorPage } from "./pages";
``` 

and in `./pages/index.js` file : 
```js
export {defalut as ErrorPage} from './ErrorPage.jsx';
```